lib/sass/gem_package_importer.rb

```diff
@@ -7,10 +7,10 @@ class GemPackageImporter
     def find_file_url(url, _canonicalize_context)
       return unless url.start_with?('pkg:')
 
-      library, _, path = url[4..].partition('/')
+      library, _, path = url[4..].partition(/[?#]/).first.partition('/')
```

Comment:
`NodePackageImporter` would throw an error complaining about querystring or hash, where `GemPackageImporter` would ignore them for simplicity. Users should not reply on this behavior and should avoid having querystring or hash.